repose-aggregator/src/docs/asciidoc/services/datastores.adoc

```diff
@@ -1,9 +1,5 @@
 = Datastores
 
-include::../_includes/in-progress.adoc[]
-
-== Configuration
-
```

Comment:
Oh yeah, because GitHub renders the markdown, so you don't get the raw source. That should totally be an option...